galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos3270.manager/src/main/java/dev/galasa/zos3270/ITerminal.java

```diff
@@ -154,13 +152,60 @@ ITerminal positionCursorToFieldContaining(@NotNull String searchText)
 
     String retrieveScreen();
 
+    /**
+     * Retrieve the contents of the field under the current cursor position.   If the field contains nulls, the string returned will 
+     * be compressed as it would be when sent back to the server.
+     * 
+     * @return The contents of the screen in the field
+     */
     String retrieveFieldAtCursor();
 
+    /**
+     * Retrieve the contents of the field under AFTER a field containing the string provided.   If the field contains nulls, the string returned will 
```

Comment:
Retrieve the contents of the field, AFTER a field containing the string provided